pkg/container/docker/exec.go

```diff
@@ -67,12 +66,6 @@ func (c *containerCmd) Run() error {
 			"-i", // interactive so we can supply input
 		)
 	}
-	// if the command is hooked up to the processes's output we want a tty
-	if log.IsTerminal(c.stderr) || log.IsTerminal(c.stdout) {
-		args = append(args,
-			"-t",
-		)
-	}
```

Comment:
minor unrelated bug, resolved in the course of working on this.

Reply:
can you please elaborate?
AFAIK this was the fix for https://github.com/kubernetes-sigs/kind/issues/659

Reply:
There's no time we need this flag. The fix for that issue was to set it less often, this sets it never.

Reply:
Elaborating: this flag allocates a pseudo tty which is nice for interactively working with a container from a shell but a) we don't do that b) it allocates the tty to both stderr and stdout c) it changes behavior around exiting (because the tty needs to be closed)